contracts/upgradeability/OwnedUpgradeabilityProxy.sol

```diff
@@ -27,25 +27,17 @@ contract OwnedUpgradeabilityProxy is UpgradeabilityOwnerStorage, UpgradeabilityP
     * @dev Throws if called by any account other than the owner.
     */
     modifier onlyProxyOwner() {
```

Comment:
It makes sense to rename `onlyProxyOwner` as well

Reply:
Done https://github.com/poanetwork/poa-bridge-contracts/pull/198/commits/f3c94b7841be3d0a4a694b30932cf4abe305f6d1